tools/ci_connector_ops/ci_connector_ops/pipelines/pipelines/metadata.py

```diff
@@ -38,6 +39,29 @@ async def _run(self) -> StepStatus:
         return await super()._run(["metadata_service", "validate", "metadata.yaml"])
 
 
+class DeployOrchestratorStep(Step):
+    def __init__(self, context: PipelineContext, title: str, parent_dir_path: str, module_path: str, dagster_cloud_api_token):
+        self.title = title
+        super().__init__(context)
+        self.parent_dir = self.context.get_repo_dir(parent_dir_path)
+        self.module_path = module_path
+        self.dagster_cloud_api_token = dagster_cloud_api_token
+
+    async def _run(self) -> StepStatus:
+        python_base = with_python_base(self.context)
+        python_with_dependencies = with_pip_packages(python_base, ["dagster-cloud==1.2.6", "poetry2setup"])
+
+        container_to_run = (
+            python_with_dependencies.with_mounted_directory("/src", self.parent_dir)
+            .with_workdir(f"/src/{self.module_path}")
+            .with_env_variable("DAGSTER_CLOUD_API_TOKEN", self.dagster_cloud_api_token)
+            .with_entrypoint(["bash"])
+            .with_exec(["deploy_to_prod.sh"])
```

Comment:
Interesting: I understand that you want to keep this script under the `metadata_service` but I think for long run maintainability we should avoid writing bash scripts. 
Do you mind splitting up the script in `with_exec` steps?

Reply:
Can do!